### DISCUSSION TITLE
Error 403 when pushing changes

### DISCUSSION THREAD

**Discussion by bionernorway** [community/community#86684] [New to GitHub]
Hi. I get this error when I want to push changes I've made for a repository:
remote: Write access to repository not granted. fatal: unable to access 'https://github.com/bioner-as/2023_norgesvel/': The requested URL returned error: 403
We recently changed the email which "owns" the organization, and after that changed the personal access token. 
Another member of the organization does not get this issue, but that's because it doesn't require the username/access token when pushing, for some reason....

**Comment by Aryma-f4** [comment]
The error message you're seeing, `remote: Write access to repository not granted`, typically indicates that the credentials you're using do not have write access to the repository. This could be due to a few reasons:

1. **Personal Access Token (PAT) Scope**: The PAT you're using might not have the correct permissions. When creating a PAT, you need to ensure it has the appropriate scope. For repository access, the 'repo' scope is required. You might need to recreate your PAT with the correct scope.

2. **Repository Settings**: Check the settings of your repository. There might be branch protection rules in place that are preventing you from pushing to the repository. 

3. **Organization Settings**: If the repository is part of an organization, there might be additional settings that are affecting your access. Check the organization's settings, particularly under 'Actions' -> 'General', and ensure that 'Actions permissions' is set to 'Allow', and 'Workflow permissions' is set to 'Read and write permissions'.

4. **SSH vs HTTPS**: Some users have reported that switching from HTTPS to SSH solved their issue. If you're currently using HTTPS, you might want to try switching to SSH.

Remember to replace `<username>:<token>` with your GitHub username and the generated PAT when you're pushing to the repository. If the problem persists, you might want to reach out to GitHub support for further assistance. Please note that these instructions are for GitHub. If you're using a different version control system, the exact steps might be different.
